packages/front-generator/src/generators/react-typescript/entity-management/template/EntityManagementEditor.tsx.ejs

```diff
@@ -109,6 +109,10 @@ class <%= editComponentName %>Component extends React.Component<Props & WrappedC
       return <Spinner/>;
     }
 
+    if (status === "ERROR") {
+      return <></>;
```

Comment:
I'd suggest returning a `null` rather than an empty fragment, it's a more common idiom.